libs/sparkpost-lib/src/test/java/com/sparkpost/EndPointTest.java

```diff
@@ -43,7 +43,7 @@ public void testSimpleEndPoint() {
         endPoint.addParam("num_rcpt_errors", 3);
 
         String result = endPoint.toString();
-        Assert.assertEquals("transmissions?num_rcpt_errors=3", result);
+        Assert.assertEquals("/transmissions?num_rcpt_errors=3", result);
```

Comment:
Should we add explicit test coverage to check that `Endpoint` manages leading slashes correctly?